client/app/models/project.js

```diff
@@ -28,26 +28,28 @@ export default class ProjectModel extends Model {
   packages;
 
   get pasPackages() {
-    const [firstPackage] = this.packages
+    const pasPackages = this.packages
       .filter((projectPackage) => projectPackage.dcpPackagetype === optionset(['package', 'type', 'code', 'PAS_PACKAGE']))
       .sortBy('dcpPackageversion')
       .reverse();
 
-    if (firstPackage) {
-      return [firstPackage];
-    }
-    return [];
+    return pasPackages;
+  }
+
+  get latestPasPackage() {
```

Comment:
Can you just quickly add to the `project.hbs` template a `Project::PackageListItem` for both the `latestPasPackage` and `latestRWCDSPackage`? I think that will help cover these lines of code.

Reply:
Sorry not sure what you mean -- we need to be listing all of the possible versions of a package on the project page and if we only pass in `latestRWCDSPackage` we'll only be getting the most recent version. Is this what you were implying?

Maybe a better fix would be to just comment out/delete these computed properties for now since they're not being used anywhere? And then we can bring them back later when we actually employ them?